fastd-exporter.go

```diff
@@ -27,6 +27,8 @@ var (
 	configPathPattern = flag.String("config-path", "/etc/fastd/%s/fastd.conf", "Override fastd config path, %s will be replaced with the fastd instance name.")
 	webListenAddress  = flag.String("web.listen-address", ":9281", "Address on which to expose metrics and web interface.")
 	webMetricsPath    = flag.String("web.telemetry-path", "/metrics", "Path under which to expose metrics.")
+	ipispEnable       = flag.Bool("ipisp.enable", true, "enable usage of ipisp lookup")
+	ipispTimeout      = flag.Int("ipisp.timeout", 300, "milliseconds to wait for ipisp lookup to finish")
```

Comment:
`ip-asn-lookup.{opt}` is more appropriate

Reply:
yes, i've changed that